src/pages/transactions/[tid].tsx

```diff
@@ -0,0 +1,735 @@
+                      "flex flex-row items-center gap-x-2.5 md:text-left lg:w-[280px] md:w-[137px] w-auto",
+                      { "md:w-[155px] w-auto gap-x-0": isToAddressCopied }
+                    )}
+                  >
+                    <div className={clsx({ "mr-[10px]": isToAddressCopied })}>
+                      <LinkText
+                        testId="transaction-token-transferred-to"
+                        label={truncateTextFromMiddle(
+                          to,
+                          windowDimension <= 1280 ? 5 : 11
+                        )}
+                        href={`/address/${to}`}
+                      />
+                    </div>
+                    <FiCopy
+                      data-testid="transaction-token-transferred-to-copy-icon"
+                      onClick={() =>
+                        onCopyAddressIconClick(setIsToAddressCopied, to)
+                      }
+                      className="text-white-50 h-[22px]"
+                    />
+                  </div>
+                )}
+              </div>
+            </div>
+            <div className="flex flex-row">
+              <div className="grow">
+                <div
+                  data-testid="transaction-token-transferred-for-title"
+                  className="text-white-700 w-[101px] tracking-[0.01em]"
+                >
+                  {fixedTitle.for}
+                </div>
+              </div>
+              <div className="flex lg:text-left md:text-left text-right">
+                <NumericFormat
+                  data-testid="transaction-token-transferred-for"
+                  className="text-white-50 tracking-[0.01em] whitespace-normal lg:w-[280px] md:w-[137px] w-auto"
+                  thousandSeparator
+                  value={forToken.value}
+                  decimalScale={8}
+                  suffix={` ${forToken.symbol}`}
+                />
+              </div>
+            </div>
+          </div>
+        )}
+      </div>
+      <div>
+        <div
+          data-testid="transaction-raw-input-title"
+          className="text-white-50 lg:pt-14 md:pt-14 pt-16 pb-6"
+        >
+          {fixedTitle.rawInput}
+        </div>
+        <div
+          className={clsx("flex flex-row items-center", {
+            "mb-[14px]": isRawInputExpanded,
+          })}
+        >
+          <div
+            data-testid="transaction-hex-title"
+            role="button"
+            tabIndex={0}
+            onKeyDown={onRawInputClick}
+            onClick={onRawInputClick}
+            className="text-white-50 pr-[10.29px]"
+          >
+            {fixedTitle.hex}
+          </div>
+          <div
+            data-testid="transaction-hex-icon"
+            role="button"
+            tabIndex={0}
+            onKeyDown={onRawInputClick}
+            onClick={onRawInputClick}
+          >
+            {isRawInputExpanded === true ? (
+              <FiChevronDown className="text-white-700" />
```

Comment:
let's increase the icon size to `24`, if you check carefully, there are "invisible" padding from the font we use vs the one in figma, so we'll need to compensate that